### DISCUSSION TITLE
[BUG][latest Version]iPhone 11 Pro Max and Android upload with browser: Figure out why Exif data is missing

### DISCUSSION THREAD

**Discussion by NetFly-VPN** [photoprism/photoprism#1338] [Q&A] [answered]
100% reproducibility
when I upload image on iPhone and android, the photo miss all exif info.
my version is the latest verison[210523-b1856b9d]

the same photo upload by compute( Mac os safari), the exif is correct

**Comment by lastzero** [comment, marked as answer]
Privacy feature? iPhone might transparently convert from HEIF to JPEG as well when uploading.

**Reply by NetFly-VPN** [reply]
No, when I upload photo from iPhone and android with browser,  all photos's exif data  is lost.

**Reply by lastzero** [reply]
See https://stackoverflow.com/questions/57942150/file-upload-and-exif-in-mobile-safari
